generator/cmd/openapi/main.go

```diff
@@ -24,23 +24,33 @@ import (
 	"github.com/googleapis/google-cloud-rust/generator/internal/genclient/translator/openapi"
 )
 
+var (
+	language      = flag.String("language", "", "the generated language")
```

Comment:
nit: any reason for moving these out of the function call? They are now globally accessible -- to this package.

Reply:
I was just following the style in:

https://github.com/googleapis/google-cloud-rust/blob/61b9d3bbac5530e4321ac19fe7d2760db82e31db/generator/devtools/cmd/generate/main.go#L27-L33

I am not sure what is right, I will leave it to you and @julieqiu to set a standard.